apollo-federation/src/sources/connect/validation/snapshots/validation_tests@denest_scalars.graphql.snap

```diff
@@ -8,7 +8,7 @@ input_file: apollo-federation/src/sources/connect/validation/test_data/denest_sc
         code: GroupSelectionRequiredForObject,
         message: "`User.street` is an object, so ``@connect(selection:)` on `Query.me`` must select a group `street{}`.",
```

Comment:
```suggestion
        message: "`User.street` is an object, so `@connect(selection:)` on `Query.me` must select a group `street{}`.",
```
Are the double backticks in this (existing) error message worth fixing?

Reply:
addressing this in the merge pr https://github.com/apollographql/router/pull/6091/commits/a0338b9065aaa749443f677b0f54bf0a9b040708